### ISSUE TITLE
New design for Repo Cards

### ISSUE THREAD

**Issue by NishidhJain** [max-programming/hacktoberfest-projects#121]

**Comment by NishidhJain** [comment]
@max-programming I would love to hear you suggestions for above design, please have a look at it. Thank You!

**Comment by max-programming** [comment]
Yeah this looks a lot more better than before. Would you or @ahmadou5 would like to take this up?

**Comment by sadiksaifi** [comment]
I would like to take this up

**Comment by sadiksaifi** [comment]
Fixes in #127

**Comment by NishidhJain** [comment]
Hi @max-programming, I would be happy to pick this up, please assign to me

**Comment by NishidhJain** [comment]
Hi @max-programming, my UI is little bit different from @sadiksaifi, have also focused on text readability and spacing issues as well. Also I have used semantic HTML tags as it helps in accessibility

@sadiksaifi New UI

![image](https://github.com/max-programming/hacktoberfest-projects/assets/61869195/59fec55a-a1d3-4b24-bd58-14616fdea8d9)

My New UI

![image](https://github.com/max-programming/hacktoberfest-projects/assets/61869195/7c92a1fb-b5f4-48f5-b9d2-85d77ae13f8d)

You will be able to notice difference is spacing, alignment & readability.

**Comment by sadiksaifi** [comment]
Do you want to continue by yourself @NishidhJain

**Comment by max-programming** [comment]
Here's what we can do. After @sadiksaifi changes are merged, you can open a new pr with your changes.

How does that sound?

**Comment by NishidhJain** [comment]
Cool @max-programming, that sounds good.

**Comment by ahmadou5** [comment]
That sound fair enough @max-programming
